container/types.proto

```diff
@@ -52,6 +52,22 @@ message Container {
   //   accepted in a NeoFS network only if the global network hashing configuration
   //   value corresponds with that attribute's value. After container inclusion, network
   //   setting is ignored.
+  // * __NEOFS__METAINFO_CONSISTENCY \
+  //   Policy rule that defines the condition under which an object is considered
+  //   processed. Acceptable values and meanings:
+  //     1. <empty>/<missing attribute>: SN does not process objects' meta
```

Comment:
attr value cannot be empty as stated above, absence is enough

Reply:
ok, reworded it to sound like "other cases" with the old behavior